app/sagas/videoConf.ts

```diff
@@ -0,0 +1,243 @@
+	}
+}
+
+function* onDirectCallJoined(payload: ICallInfo) {
+	const calls = yield* appSelector(state => state.videoConf.calls);
+	const currentCall = calls.find(c => c.callId === payload.callId);
+	if (currentCall && (currentCall.action === 'accepted' || currentCall.action === 'calling')) {
+		yield put(setCalling(false));
+		yield put(removeVideoConfCall(currentCall));
+		yield call(hideActionSheetRef);
+		videoConfJoin(payload.callId, false, true);
+	}
+}
+
+function* onDirectCallEnded(payload: ICallInfo) {
+	const calls = yield* appSelector(state => state.videoConf.calls);
+	const currentCall = calls.find(c => c.callId === payload.callId);
+	if (currentCall) {
+		yield put(removeVideoConfCall(currentCall));
+		hideCustomNotification();
+	}
+}
+
+function* handleVideoConfIncomingWebsocketMessages({ data }: { data: any }) {
+	const { action, params } = data.action;
+
+	if (!action || typeof action !== 'string') {
+		return;
+	}
+	if (!params || typeof params !== 'object' || !params.callId || !params.uid || !params.rid) {
+		return;
+	}
+	const prop = { ...params, action };
+	switch (action) {
+		case 'call':
+			yield call(onDirectCall, prop);
+			break;
+		case 'canceled':
+			yield call(onDirectCallCanceled, prop);
+			break;
+		case 'accepted':
+			yield call(onDirectCallAccepted, prop);
+			break;
+		case 'rejected':
+			yield call(onDirectCallRejected, prop);
+			break;
+		case 'confirmed':
+			yield call(onDirectCallConfirmed, prop);
+			break;
+		case 'join':
+			yield call(onDirectCallJoined, prop);
+			break;
+		case 'end':
+			yield call(onDirectCallEnded, prop);
+			break;
+	}
+}
+
+function* initCall({ payload: { mic, cam, direct, rid } }: { payload: TCallProps }) {
+	yield put(setCalling(true));
+	const serverVersion = yield* appSelector(state => state.server.version);
+	const isServer5OrNewer = compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '5.0.0');
+	if (isServer5OrNewer) {
+		try {
+			const videoConfResponse = yield* call(Services.videoConferenceStart, rid);
+			if (videoConfResponse.success) {
+				if (direct && videoConfResponse.data.type === 'direct') {
+					yield call(callUser, { rid, uid: videoConfResponse.data.calleeId, callId: videoConfResponse.data.callId });
+				} else {
+					videoConfJoin(videoConfResponse.data.callId, cam, mic);
+					yield call(hideActionSheetRef);
+					yield put(setCalling(false));
+				}
+			}
+		} catch (e) {
+			yield put(setCalling(false));
+			showErrorAlert(i18n.t('error-init-video-conf'));
+			log(e);
+		}
+	} else {
+		const sub = yield* call(getSubscriptionByRoomId, rid);
+		if (sub) {
+			callJitsi({ room: sub, cam });
+			yield put(setCalling(false));
+		}
+	}
+}
+
+function* giveUp({ rid, uid, callId, rejected }: { rid: string; uid: string; callId: string; rejected?: boolean }) {
+	yield put(removeVideoConfCall({ rid, uid, callId }));
+	notifyUser(`${uid}/video-conference`, { action: rejected ? 'rejected' : 'canceled', params: { uid, rid, callId } });
```

Comment:
at line 74 you are calling `yield call(notifyUser, ...`, here you are using a different way and as not yield the function will be treated as sync, right? Can use the same way for both?